src/Features/Onboarding.php

```diff
@@ -765,6 +765,7 @@ public static function get_onboarding_allowed_plugins( $plugins ) {
 				'woo-razorpay'                        => 'woo-razorpay/woo-razorpay.php',
 				'mollie-payments-for-woocommerce'     => 'mollie-payments-for-woocommerce/mollie-payments-for-woocommerce.php',
 				'payu-india'                          => 'payu-india/index.php',
+				'mailpoet'                            => 'mailpoet/mailpoet.php',
```

Comment:
This might have to be `mailpoet/index.php`, as I don't see `mailpoet.php` when I install it locally, although it does seem to exist on svn: https://plugins.trac.wordpress.org/browser/mailpoet/tags/3.60.2 
I am not a 100% sure if we actually rely on these path's though, I don't think we do.

Reply:
Mmm, that's interesting 🤔 
I can see the `mailpoet.php` file in my installation.
On the other hand, I think that replacing the route `mailpoet/mailpoet.php` with `mailpoet/index.php` breaks the activation process for MailPoet in the OBW.